src/adr/articles/0006-openapi.md

```diff
@@ -0,0 +1,69 @@
+---
+layout: layouts/adr.njk
+title: "OpenAPI Specifications"
+weight: 1
+date: 2022-02-01
+review_in: 12 months
+tags: adr
+areas_of_coverage: ["Digital Service"]
+status: "accepted"
+contributors: ["John Nolan"]
+adr_number: "0006"
+---
+
+## Context
+
+We need a way to communicate our APIs intent to consumers. We should display all options available as well as required data and validation associated with integrating with our APIs.
+
+Any changes to the schema of our API should be testable via this specification.
+
+## Technical
+
+### Interoperability - How does this enable the exchange of information
```

Comment:
Worth referencing the https://www.gov.uk/government/collections/api-design-guidance

Reply:
Added and linked that reference